src/main/c/zmq/zmq_pub.cc

```diff
@@ -197,7 +197,7 @@ int main (void) {
   char message_ch[120];
   while(true) {
     zmq_pollitem_t items[] = {
```

Comment:
I guess if we just install libzmq3-dev and do not copy zmq.hpp of our own, we should be fine.
Let me double check it.